package.json

```diff
@@ -27,7 +27,7 @@
     "@types/express": "^4.0.33",
     "@types/mime": "0.0.29",
     "@types/mz": "0.0.29",
-    "@types/node": "^4.0.30",
+    "@types/node": "=4.0.30",
```

Comment:
Turns out my error had nothing to do with this change :)

It was actually caused by Node 4 not supporting the ES6 spread operator.